tox.ini

```diff
@@ -1,13 +1,10 @@
 [tox]
 envlist =
-    py26,py27,py33
+    py27,py33,py34,py35,pypy
 
 [testenv]
 commands =
     python setup.py test -q
 deps =
-    z3c.pt
-    zope.configuration
-    zope.pagetemplate
-    zope.testing
-    zope.traversing
+    .[test]
+    z3c.pt >= 3.0.0a1
```

Comment:
Otherwise the tests will fail under Python 3 (see [Travis](https://travis-ci.org/zopefoundation/z3c.ptcompat/jobs/40151146)) because tox only uses non-final versions if it is forced to.